web/server/codechecker_server/api/report_server.py

```diff
@@ -91,17 +93,23 @@ def comment_kind_from_thrift_type(kind):
     """ Convert the given comment kind from Thrift type to Python enum. """
     if kind == ttypes.CommentKind.USER:
         return CommentKindValue.USER
-    elif kind == ttypes.CommentKind.SYSTEM:
+
+    if kind == ttypes.CommentKind.SYSTEM:
         return CommentKindValue.SYSTEM
 
+    assert False, f"Unknown ttypes.CommentKind: {kind}"
+
 
 def comment_kind_to_thrift_type(kind):
     """ Convert the given comment kind from Python enum to Thrift type. """
     if kind == CommentKindValue.USER:
         return ttypes.CommentKind.USER
-    elif kind == CommentKindValue.SYSTEM:
+
+    if kind == CommentKindValue.SYSTEM:
```

Comment:
Courageously, we can use elif statement.

Reply:
Same answer.